airbyte-integrations/connectors/source-outreach/source_outreach/schemas/call_purposes.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": "integer"
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "order": {
+      "type": ["null", "integer"]
+    },
+    "createdAt": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "updatedAt": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "creator": {
```

Comment:
i think our API doesn't have data in these arrays but we're following how other fields are implementing here, correct?

Reply:
In the response samples in the API docs, i saw that there was a creator and call objects: https://developers.outreach.io/api/reference/tag/Call-Purpose/#tag/Call-Purpose/paths/~1callPurposes/get

But yeah in the records returned from the read command, those arrays are empty

Reply:
Nice, let's continue using what we have today. Thanks Saj!